StPetersburg/InEtn210322.xml

```diff
@@ -0,0 +1,200 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0"
+     xml:id="InEtn210321"
+     xml:lang="en"
+     type="mss">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title>Collection of Magic Texts</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DE"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens: Eine multimediale
+                            Forschungsumgebung / beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+              <msDesc type="scroll">
+                <msIdentifier>
+                    <repository ref="INS0524InEtn"/>
+                    <idno>2103-22</idno>
+                    <altIdentifier>
+                      <idno>Platonov cat. III.115</idno>
+                    </altIdentifier>
+                </msIdentifier>
+                <msContents>
+                  <summary>
+                  </summary>
+                  <msItem xml:id="ms_i1">
+                    <title type="complete" ref="LIT5844MagicPrayer" xml:lang="gez">ጸሎት፡ በእንተ፡ ዛር፡</title>
+                    <textLang mainLang="gez"/>
+                    <note>Lines 1-24.</note>
+                  </msItem>
+                  <msItem xml:id="ms_i2">
+                    <title type="complete" ref="LIT5077MagicPrayer" cert="medium" xml:lang="gez">ጸሎት፡ በእንተ፡ ዓይነ፡ ጥላ፡</title>
```

Comment:
cert="medium" is not very meaningful in itself. Is the identification of the work with the LIT record somewhat uncertain?

Reply:
yes I have corrected the expression